### PR TITLE
Navbar is now fixed on screen and Song page's song details change when user clicks on play button

### PR DESCRIPTION
**Pull request by lchua2314**
- Navbar is now fixed on screen
- Song page's song details change when user clicks on play button of a different song
- Refactored Song page's API calls
- Converted Song.css into a CSS Module to fix CSS conflicts

### CODE REVIEW

**Review by leonzh2k — APPROVED** [review]
So the song details next to the image change when you play a track? Also, just curious, how does the fixed navbar look with different screen sizes? Otherwise seems fine.

**Comment by lchua2314** [comment]
Yeah, only the song name changes since all the songs are in the same playlist. Napster did it so I thought it'll be a good feature. As for the navbar, I think it's fine on different screen sizes, but I wouldn't really know until we test it on an actual mobile device.

**Comment by lchua2314** [comment]
New commits:
- Refactored Song page's API calls
- Converted Song.css into a CSS Module to fix CSS conflicts

**Review by leonzh2k — COMMENTED** [review]
Since the async functions are pretty similar, maybe you could put all of them in the same file so there aren't so many import statements.

**Comment by lchua2314** [comment]
I was thinking of doing that, but how would I specify which function to call if that's even possible?

**Comment by leonzh2k** [comment]
> I was thinking of doing that, but how would I specify which function to call if that's even possible?

Some answers: https://stackoverflow.com/questions/49616639/how-can-i-export-all-functions-from-a-file-in-js

**Comment by lchua2314** [comment]
Got it to work. Are the namings good?

**Review by leonzh2k — APPROVED** [review]
> Got it to work. Are the namings good?

Looks good

File: src/modules/Song/Song.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { useState } from 'react';
-import './Song.css';
+import styles from './Song.module.css';
```
[the remark this answers is not in the document]
Comment (lchua2314) [reply src/modules/Song/Song.js:3]: I tried without it first. It didn't work.

[the remark this answers is not in the document]
Comment (leonzh2k) [reply src/modules/Song/Song.js:3]: How did you test it to find out?

**Review by roundband5 — APPROVED** [review]
good

**Pull request merged**